isisdaeApp/Db/isisdae.db

```diff
@@ -372,6 +372,39 @@ record(ai, "$(P)$(Q)EVENTMODEFRACTION")
 	field(EGU, "")
 }
 
+record(ai, "$(P)$(Q)EVENTMODEBUFUSED")
+{
+    field(DESC, "Event Mode Buffer used Fraction")
+    field(DTYP, "asynFloat64")
+    field(INP,  "@asyn(icp,0,0)EVENTMODEBUFFERUSED")
+    field(PREC, "3")
+    field(SCAN, "I/O Intr")
+    info(INTEREST, "HIGH")
+	field(EGU, "")
+}
+
+record(ai, "$(P)$(Q)EVENTMODEFILEMB")
+{
+    field(DESC, "Event Mode File MB")
+    field(DTYP, "asynFloat64")
+    field(INP,  "@asyn(icp,0,0)EVENTMODEFILEMB")
+    field(PREC, "3")
+    field(SCAN, "I/O Intr")
+    info(INTEREST, "HIGH")
+	field(EGU, "")
```

Comment:
I presume MB refers to a file size, so this should probably have units?

Reply:
Indeed - "MB" added to EGU field